plugins/tiddlywiki/highlight/files/tiddlywiki.files

```diff
@@ -2,6 +2,16 @@
 	"tiddlers": [
 		{
 			"file": "highlight.pack.js",
+			"fields": {
+				"type": "application/javascript",
+				"title": "$:/plugins/tiddlywiki/highlight/highlight-9.js",
+				"module-type": "library"
+			},
+			"prefix": "var hljs = require(\"$:/plugins/tiddlywiki/highlight/highlight-9.js\");\n",
+			"suffix": "\nexports.hljs = hljs;\n"
```

Comment:
Why do you include both versions? ... 

IMO only 1 version should be part of the plugin!

Reply:
> Since highlight.js is now using ES2015, I added conditional require to support both modern and legacy browsers. If this is not necessary, let me know and I can remove old dependencies. Legacy browsers tested: iOS9 on ipad3 and IE11.

I wasn't quite sure if TW is staying with ES5 as long as it can--as 5.2.1 still loads nicely on my ipad3--and I think highlight.js is quite useful, so I included both versions for backward compatibility. Obviously this increases the size of the plugin...

I can remove v9 easily.

Reply:
We are indeed trying to stick with ~ES5.x as long as possible, so it is certainly useful to have the older version available. I wonder if it might be easier to make them be entirely different plugins, offered alongside one another in the plugin library.